src/doc/gui_manual/Operators/OperatorTypes/PICS/IntegralCurve_operator.rst

```diff
@@ -0,0 +1,298 @@
+    * X Extents - extents along the X axis.
+    * Y Extents - extents along the Y axis.
+    * Z Extents - extents along the Z axis. 
+    * Sampling - Specify the type of sampling, the region from which to sample (interior or boundary) and the number of samples.
+   
+Selection
+    Seed with a named selection.
+    
+Field Data
+    The seed points are defined by another operator and passed to the Integral Curve operator. The array containing the seed point(s) must begin its name with "Seed Points".
+
+    * Copy to point list - Copy the points to the point list source (see above). This allows points to be edited. 
+
+
+Sampling type
+"""""""""""""
+
+For samples taken from a line, circle, plane, sphere or box, there is an option to generate uniform or random samples from the specified region. In more detail:
+
+Uniform
+    Sample uniformly along the boundary or the interior. 
+
+    * Samples in X - (Default 1) Create N samples along the X axis.
+    * Samples in Y - (Default 1) Create N samples along the Y axis.
+    * Samples in Z - (Default 1) Create N samples along the Z axis. 
+   
+Random
+    Sample randomly along the boundary or the interior. 
+
+    * Random number of samples (Default 1).
+    * Random number Seed - (Default 0) ensures that the same set of random samples is taken each time the plot is regenerated. 
+
+Boundary vs Interior Samples
+""""""""""""""""""""""""""""
+
+Samples from a circle, plane, sphere, or box can be either taken from the boundary, or the interior. For example, in the case of a plane sampling source, the samples can either lie along the edges of the planar region, or within the bounded rectangle, as shown below.
+    
+Field
+"""""
+
+See Field attributes that are common to all PICS operators.
+
+
+Integration
+^^^^^^^^^^^
+
+Specify settings for numerical integrators. In addition to the Integration attributes common to all PICS operators, the Integral Curve operator supports the following attributes.
+
+Integration Direction
+"""""""""""""""""""""
+
+Sets the integration direction through time. Options are:
+    
+Forward
+    Integrate forward in time.
+
+Backward
+    Integrate backward in time.
+
+Both
+    Integrate both forward and backward in time, producing two integral curves.
+
+Forward Directionless
+    Integrate forward in time assuming a directionless vector field.
+
+Backward Directionless
+    Integrate backward in time assuming a directionless vector field.
+
+Both Directionless
+    Integrate both forward and backward in time assuming a directionless vector field, producing two integral curves.
+
+Eigen vectors are an example of a directionless vector field. To integrate using a directionless field requires that any orientation discontinuity be corrected prior to linear interpolation. That is, all vectors must be rotated to match the orientation of the trajectory. The PICS code will do this processing for standard fields (e.g non-higher order elements).
```

Comment:
I don't understand what a *directionless* vector field is. This juxtoposition of *directionless* and *vector* confuses me because vectors *have* magnitude *and* direction. I think this might be getting at whether the vector field is *continuous* in direction.

Reply:
I don't understand what directionless is either... I pulled this explanation off the original documentation in visit users. I definitely need to look into this more.